politeiawww/politeiawww.go

```diff
@@ -1076,3 +1076,52 @@ func (p *politeiawww) setPoliteiaWWWRoutes() {
 	p.addRoute(http.MethodPost, v1.RouteCensorComment,
```

Comment:
This should go into cmswww.go. That makes it symmetric with userwww.go and politeiawwww.go